Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 1 (use --cores to define parallelism)
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	2	clean_fasta
	6	concat_fasta
	2	concat_indelible
	3	concat_summaries
	1	create_control_files
	1	create_plots
	6	run_grasp
	2	run_indelible
	6	summarise_indels
	29
Select jobs to execute...

[Sun Jan 31 14:51:16 2021]
rule create_control_files:
    output: indelible/8_control.txt
    jobid: 6
    wildcards: taxon=8

[Sun Jan 31 14:51:17 2021]
Finished job 6.
1 of 29 steps (3%) done
Select jobs to execute...

[Sun Jan 31 14:51:17 2021]
rule run_indelible:
    input: indelible/8_control.txt
    output: indelible_output/8/cleaned_trees/1.nwk, indelible_output/8/8_TRUE_1.fasta, indelible_output/8/8_ANCESTRAL_1.fasta
    jobid: 5
    wildcards: taxon=8, rep=1

[Sun Jan 31 14:51:17 2021]
Finished job 5.
2 of 29 steps (7%) done
Select jobs to execute...

[Sun Jan 31 14:51:17 2021]
rule run_indelible:
    input: indelible/8_control.txt
    output: indelible_output/8/cleaned_trees/2.nwk, indelible_output/8/8_TRUE_2.fasta, indelible_output/8/8_ANCESTRAL_2.fasta
    jobid: 12
    wildcards: taxon=8, rep=2

Warning: the following output files of rule run_indelible were not present when the DAG was created:
{'indelible_output/8/cleaned_trees/2.nwk', 'indelible_output/8/8_ANCESTRAL_2.fasta', 'indelible_output/8/8_TRUE_2.fasta'}
[Sun Jan 31 14:51:18 2021]
Finished job 12.
3 of 29 steps (10%) done
Select jobs to execute...

[Sun Jan 31 14:51:18 2021]
rule clean_fasta:
    input: indelible_output/8/8_TRUE_2.fasta
    output: indelible_output/8/cleaned_aln/2.fasta
    jobid: 11
    wildcards: taxon=8, rep=2

[Sun Jan 31 14:51:18 2021]
Error in rule clean_fasta:
    jobid: 11
    output: indelible_output/8/cleaned_aln/2.fasta

RuleException:
CalledProcessError in line 32 of /Users/gabefoley/Dropbox/PhD/20210121_Indel_Evaluation_Project/Indel_Evaluation_Snakemake/Snakefile:
Command 'set -euo pipefail;  /Users/gabefoley/miniconda3_latest/envs/indel_snakemake/bin/python3.9 /Users/gabefoley/Dropbox/PhD/20210121_Indel_Evaluation_Project/Indel_Evaluation_Snakemake/.snakemake/scripts/tmpf7e4ljh1.clean_fasta.py' returned non-zero exit status 1.
  File "/Users/gabefoley/miniconda3_latest/envs/indel_snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 2319, in run_wrapper
  File "/Users/gabefoley/Dropbox/PhD/20210121_Indel_Evaluation_Project/Indel_Evaluation_Snakemake/Snakefile", line 32, in __rule_clean_fasta
  File "/Users/gabefoley/miniconda3_latest/envs/indel_snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 568, in _callback
  File "/Users/gabefoley/miniconda3_latest/envs/indel_snakemake/lib/python3.9/concurrent/futures/thread.py", line 52, in run
  File "/Users/gabefoley/miniconda3_latest/envs/indel_snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 554, in cached_or_run
  File "/Users/gabefoley/miniconda3_latest/envs/indel_snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 2350, in run_wrapper
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /Users/gabefoley/Dropbox/PhD/20210121_Indel_Evaluation_Project/Indel_Evaluation_Snakemake/.snakemake/log/2021-01-31T145116.607756.snakemake.log
